### DISCUSSION TITLE
Messaggi di notifica continui

### DISCUSSION THREAD

**Discussion by LuBian69** [globaleaks/globaleaks-whistleblowing-software#4203] [General]
Buongiorno,
è da ieri che continuiamo a ricevere sull'indirizzo delle notifiche una quantità enorme di segnalazioni che non corrispondono a nessuna richiesta. Il messaggio della mail è il seguente:

Platform: Whistleblowing - Ministero dell'Interno - Modulo per la segnalazione di condotte illecite  ai sensi del decreto legislativo n. 24/2023
Host: wbsegnalazioni.interno.gov.it (336lvyhkw5nry66bgkllf66ksg52avkrkuf6lao55akqvvkmsg2s24qd.onion)
Version: 4.11.5

twisted.web._newclient.ResponseNeverReceived 
    A L{ResponseFailed} that knows no response bytes at all have been received.
    

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/globaleaks/jobs/job.py", line 49, in run
    yield self.operation()

twisted.web._newclient.ResponseNeverReceived: [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', '', 'unexpected eof while reading')]>]

**Comment by LuBian69** [comment]
Versione del software: 4.11.5

**Comment by evilaliv3** [comment]
Grazie per la domanda @LuBian69 

Questi suppongo sono messaggi che ricevete come amministratore sulla vostra istanza; probabile che le vostre regole di firewall stiano bloccando le connessioni in uscita verso i nostri server che il sistema effettua per verificare la disponibilità di aggiornamenti.

Con che frequenza ricevi le email ? ogni ora / mezzora?
